### PR TITLE
Take config through env.

### PR DESCRIPTION
**Pull request by arijitAD**
fixes:https://github.com/vulcanize/ipld-eth-server/issues/119

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -121,14 +124,21 @@ func init() {
...
+	}
+
+	viper.SetConfigFile(cfgFile)
+	if err := viper.ReadInConfig(); err != nil {
+		log.Fatalf("Couldn't read config file: %s", err.Error())
+	}
+
+	log.Infof("Using config file: %s", viper.ConfigFileUsed())
+
+	if filepath.Ext(cfgFile) == ".env" {
```
Comment (i-norden) [inline cmd/root.go:139]: What if we want to load some params from a .toml and others from a .env? These files won't necessarily be the same, I think we should introduce a new CLI flag for specifying a ENV file path distinct from the TOML config file path. What do you think?

**Review by i-norden — COMMENTED** [review]
One comment/question:

File: cmd/root.go (hunk 1 above)
Comment (arijitAD) [reply cmd/root.go:139]: I will update the code to use a different flag for CLI.

**Review by i-norden — APPROVED** [review]
LGTM, thanks for making the update

**Pull request merged**